src/jupyterscad/_openscad.py

```diff
@@ -0,0 +1,85 @@
+"""
+Jupyter SCAD
+Copyright (C) 2023 Jennifer Reiber Kyle
+
+This program is free software: you can redistribute it and/or modify it under
+the terms of the GNU General Public License as published by the Free Software
+Foundation, either version 3 of the License, or (at your option) any later
+version.
+
+This program is distributed in the hope that it will be useful, but WITHOUT ANY
+WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+PARTICULAR PURPOSE. See the GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License along with
+this program. If not, see <https://www.gnu.org/licenses/>.
+"""
+import logging
+import platform
+import subprocess
+from pathlib import Path
+from shutil import which
+
+from .exceptions import OpenSCADException
+
+LOGGER = logging.getLogger(__name__)
+
+DEFAULT_OPENSCAD_EXECUTABLE = {
+    "Linux": ["/usr/bin/openscad", "/usr/local/bin/openscad"],
```

Comment:
I think this is redundant, because I would assume that `/usr/bin` and `/usr/local/bin` are supposed to be in `PATH` on every(? at least on every more or less regular)  linux system and therefor `which` should pick it up anyway beforehand.

Furthermore if `/usr/bin/` (and `.../local/...`) are not in `PATH`, I think the chance of finding an `openscad` executable in that directory are pretty low.

Reply:
Seems to be working....

Reply:
> Does `shutil.which` pick up your openscad executable (on macos, right?)?

not as installed, no. It does if I set up the machine to add the openscad executable directory to path. I'm testing with a windows VM and it looks like `shutil.which` doesn't pick up the executable on Windows either.

thanks for the heads up on the linux machine. I'll drop the custom executable locations for linux and keep the one for mac.